internal/pkg/composable/providers/kubernetessecrets/kubernetes_secrets.go

```diff
@@ -175,11 +176,13 @@ func (p *contextProviderK8SSecrets) Fetch(key string) (string, bool) {
 	p.store.AddConditionally(key, sd, true, func(existing secret, exists bool) bool {
 		if !exists {
 			// no existing secret in the cache thus add it
+			p.logger.Infof("Fetch: \"%s\" inserted. Resource Version of secret: \"%s\"", key, apiSecretResourceVersion)
```

Comment:
Why not the following?

```suggestion
			p.logger.Infof(`Fetch: "%s" inserted. Resource Version of secret: "%s"`, key, apiSecretResourceVersion)
```